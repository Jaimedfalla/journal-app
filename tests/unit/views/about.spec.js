import AboutView from '@/views/AboutView.vue'
import { shallowMount } from '@vue/test-utils';

describe('About view component',()=>{
    let wrapper;

    beforeEach(()=>{
        wrapper = shallowMount(AboutView)
    })

    it("Should match with snaptshon",()=>{
        expect(wrapper.html()).toMatchSnapshot()
    })
})