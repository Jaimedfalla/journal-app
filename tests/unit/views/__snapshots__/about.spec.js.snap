// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About view component Should match with snaptshon 1`] = `
<div class="about">
  <h1>This is an about page</h1>
</div>
`;
